mysite/lti/models.py

```diff
@@ -0,0 +1,94 @@
+import json
+from django.db import models
+from django.contrib.auth import login
+from django.contrib.auth.models import User
+
+from social.apps.django_app.default.models import UserSocialAuth
+
+from ct.models import Role, Course
+
+
+class LTIUser(models.Model):
+    """Model for LTI user
+
+    Created for Moodle LMS.
```

Comment:
Where exactly it is Moodle specific?
Please describe extra_Data and consumer field in docsting